backend/src/main/java/com/eyetrade/backend/controller/CurrencyController.java

```diff
@@ -13,17 +15,19 @@
  * Created by Emir Gökdemir
  * on 17 Eki 2019
  */
-@Api(value = "Currencies")
+@Api(value = "Currencies", tags = {"Operations Related With Currencies"})
 @RestController
 @RequestMapping("/currency")
 public class CurrencyController {
 
     @Autowired
     CurrencyConverterService currencyConverterService;
 
+    @ApiOperation(value = "Takes the source currency, target currency, amount of source currency; then converts the currency amount", response = CurrencyConverterResource.class)
     @GetMapping("/convert")
-    public CurrencyConverterResource convertCurrency(CurrencyConverterDto converterDto){
-        return currencyConverterService.convertCurrency(converterDto);
```

Comment:
There is no differrence between this two types, because Spring handle Response code by @Getmapping annotation.

Reply:
I have changed it because of the convention but if the team prefer the other approach we can change the convention.

Reply:
No problem in my opinion, from now onward, we will use ResponseEntity approach.